src/views/document/utils/document-view-mixin.js

```diff
@@ -95,9 +95,11 @@ export default {
      * properties that are deducted from URL
      */
     documentId() {
+      if (this.isDraftView || this.isPrintingView) return this.draft.document_id;
```

Comment:
```suggestion
      if (this.isDraftView || this.isPrintingView) {
        return this.draft.document_id;
      }
```

We try to avoid such shortcut notations in the code, for readablitity.